### PR TITLE
OSDOCS-9882-authentication: Documented the authentication 4.16 bug fi…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16

Issue:
[OSDOCS-9882](https://issues.redhat.com/browse/OSDOCS-9882)

Link to docs preview:
[Bug fixes - API Server and Authentication](https://77731--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-bug-fixes_release-notes)

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -1083,6 +1083,12 @@ Kubernetes 1.29 removed the following deprecated APIs, so you must migrate manif
 [id="ocp-4-16-api-auth-bug-fixes_{context}"]
 ==== API Server and Authentication
 
+* Previously, `ephemeral` and `csi` volumes were not properly added to security context constraints (SCCs) on upgraded clusters. With this release, SCCs on upgraded clusters are properly updated to have `ephemeral` and `csi` volumes. (link:https://issues.redhat.com/browse/OCPBUGS-33522[*OCPBUGS-33522*])
+
+* Previously, the `ServiceAccounts` resource could not be used with OAuth clients for a cluster with the `ImageRegistry` capability enabled . With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-30319[*OCPBUGS-30319*])
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:1088]:
```suggestion
* Previously, the `ServiceAccounts` resource could not be used with OAuth clients for a cluster with the `ImageRegistry` capability enabled. With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-30319[*OCPBUGS-30319*])
```

**Comment by mburke5678** [comment]
@dfitzmau One nit. Otherwise LGTM

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**